src/dwd_dynam.jl

```diff
@@ -381,7 +469,7 @@ end
 
 ### Helper functions for `oapply`
 
-function induced_dynamics(d::WiringDiagram, ms::Vector{M}, S) where {T,I, M<:AbstractMachine{T,I}}
+function induced_dynamics(d::WiringDiagram, ms::Vector{M}, S, get_readouts = get_readouts) where {T,I, M<:AbstractMachine{T,I}}
```

Comment:
mentioned in my PR #100 but this method (and induced_readout) might be too parametrically constrained!